src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -1177,6 +1201,16 @@ static ssize_t status_show(struct device *dev,
 }
 static DEVICE_ATTR_RO(status);
 
+static ssize_t core_version_show(struct device *dev,
+	struct device_attribute *da, char *buf)
+{
+	struct xocl_xmc *xmc = dev_get_drvdata(dev);
+	u32 val = READ_REG32(xmc, XMC_CORE_VERSION_REG);
+
+	return sprintf(buf, "0x%x\n", val);
```

Comment:
do you want to show the interpreted version like x.y.z, which is more readable?

Reply:
Yes, I just changed to x.y.z format.

cat /sys/bus/pci/drivers/xclmgmt/0000\:da\:00.1/xmc.m.12582916/core_version
1.0.4